src/bootstrap/src/core/build_steps/vendor.rs

```diff
@@ -1,9 +1,14 @@
+//! Handles the vendoring process for the bootstrap system.
+//!
+//! This module ensures that all required Cargo dependencies are gathered
+//! and stored in the `<src>/VENDOR_DIR` directory.
```

Comment:
```suggestion
//! and stored in the `<src>/<VENDOR_DIR>` directory.
```
Nit: just to avoid making it look like a constant.